docker-compose/jaeger/docker-compose.yaml

```diff
@@ -0,0 +1,45 @@
+version: '2.4'
+services:
+  # Description: Jaeger all-in-one instance
+  # (https://www.jaegertracing.io/docs/1.17/getting-started/#all-in-one) for distributed tracing.
+  #
+  # Disk: none
+  # Ports exposed to other Sourcegraph services: 5778/TCP 6831/UDP 6832/UDP 14250/TCP
+  # Ports exposed to the public internet: none
+  # Ports exposed to site admins only: 16686/HTTP
+  #
+  jaeger:
+    container_name: jaeger
+    image: 'index.docker.io/sourcegraph/jaeger-all-in-one:169594_2022-08-27_c479e442c579@sha256:eed16144aada80f20e72c86c67ed89ba3aca109cfac25f4fe68b7fc9a0588414'
+    cpus: 0.5
+    mem_limit: '512m'
+    ports:
+      # Query port
+      - '0.0.0.0:16686:16686'
+      # Collector port
+      - '0.0.0.0:14250:14250'
+      # Agent ports
+      - '0.0.0.0:5778:5778'
+      - '0.0.0.0:6831:6831'
+      - '0.0.0.0:6832:6832'
+    networks:
+      - sourcegraph
+    restart: always
+    command: ['--memory.max-traces=20000']
+    environment:
+      - 'SAMPLING_STRATEGIES_FILE=/etc/jaeger/sampling_strategies.json'
+
+  # Configure collector to send traces to Jaeger
```

Comment:
```suggestion
  # Configure collector to send traces to Jaeger
  # By default listens on the following ports:
  # - grpc 4317
  # - http 4318
```

Reply:
This is already indicated on docstring for the underlying service configuration in the base `docker-compose.yaml` file, so I think we should be fine to omit this here